client/src/utils/fakeCardData.js

```diff
@@ -4,80 +4,80 @@ export const cardData = [
     body:
       'This virus was first detected in Wuhan City, Hubei Province, China. The first infections were linked to a live animal market, but the virus is now spreading from person-to-person.',
     img: 'https://upload.wikimedia.org/wikipedia/en/f/f5/RWS_Tarot_08_Strength.jpg',
-    footer: 'fun fact# 1',
+    footer: 'fact# 1',
```

Comment:
```suggestion
    footer: 'Fact 1',
```

Sorry I didn't notice [your reply](https://github.com/COVID-19-electronic-health-system/Corona-tracker/issues/645#issuecomment-619467960) sooner, but I agree. Let's remove the `#` and capitalize `fact` for all of the fact footers.